lib/fizzy/execute.cpp

```diff
@@ -409,6 +409,7 @@ inline void comparison_op(Stack<uint64_t>& stack, Op<T> op) noexcept
 template <typename T>
 inline T shift_left(T lhs, T rhs) noexcept
 {
+    static_assert(!std::numeric_limits<T>::is_signed);
```

Comment:
This may be a bit more precise.
```suggestion
    static_assert(std::is_unsigned_v<T>);
```